### PR TITLE
 Add setup IoT Edge Simulator command

### PR DESCRIPTION
**Pull request by LazarusX**

### CODE REVIEW

**Comment by adashen** [comment]
Fix lint errors

File: src/common/utility.ts [hunk 1]
```diff
@@ -398,6 +399,21 @@ export class Utility {
         }
     }
 
+    public static async getInputDevice(deviceItem: DeviceItem, outputChannel: vscode.OutputChannel, onlyEdgeDevice: boolean = true): Promise<DeviceItem> {
```
Comment (formulahendry) [inline src/common/utility.ts]: Actually, `getDevice` does not return DeviceItem...

Comment (LazarusX) [reply src/common/utility.ts]: Fixed [here](https://github.com/Microsoft/vscode-azure-iot-edge/pull/202/commits/bff01c73524cf99dda2e9b44aa1136ff6c1fcaa1#diff-0ed8845dd6e96c3bb0edd1befb3aa5db).

**Comment by LazarusX** [comment]
@adashen Linter issue fixed

**Pull request merged**